README.md

```diff
@@ -685,8 +685,8 @@ New API version v4.0 is now available in Production
   the `calculationType`, `totalIncomeTaxAndNicsDue` is now optional and
   `calculationTimeStamp` now has three digits for milliseconds instead of two
 * `Retrieve a Self Assessment Tax Calculation` has new
-  properties `totalAnnualAllowancesAndDeductions`, `totalBasicRateExtension`, `giftAidRelief`, `pensionsContributionRelief`
-  and `offPayrollWorker`
+  properties `calculation.endOfYearEstimate.totalAnnualAllowancesAndDeductions`, `calculation.reliefs.basicRateExtension.totalBasicRateExtension`, `calculation.reliefs.basicRateExtension.giftAidRelief`, `calculation.reliefs.basicRateExtension.pensionsContributionRelief`
+  and `calculation.employmentAndPensionsIncomeDetail.offPayrollWorker`
```

Comment:
In the tech spec and OAS, the field name for `totalAnnualAllowancesAndDeductions` is `calculation.endOfYearEstimate.totalAllowancesAndDeductions`, and the path for `offPayrollWorker` is `calculation.employmentAndPensionsIncome.employmentAndPensionsIncomeDetail.offPayrollWorker`